<launch>
  <node pkg="faceTracking" type="faceTrack" name="flow" output="screen">
	<remap from="image" to="/rgb/image_color"/>
  </node>

  <node pkg="openni_camera" type="openni_node" name="kinect">
	  <param name="/image_mode" value="2"/>
  </node>


  <node pkg="image_proc" type="image_proc" name="bayer2rgb">
	<env name="ROS_NAMESPACE" value="/rgb"/>
  </node>

  <node pkg="image_view" type="image_view" name="im_in">
	<remap from="image" to="/rgb/image_color"/>
  </node>

  <node pkg="image_view" type="image_view" name="im_out1">
	<remap from="image" to="/handImage"/>
  </node>

  <node pkg="image_view" type="image_view" name="im_out">
	<remap from="image" to="/outputImage"/>
  </node>

  <node pkg="handBlobTracker" type="handTracker" name="handTracker" output="screen"/>

  <node pkg="pfBodyTracker" type="poseTracker" name="poseTracker" output="screen"/>

  <node pkg="image_view" type="image_view" name="im_out2">
        <remap from="image" to="/poseImage"/>
  </node>


</launch>
